package.json

```diff
@@ -240,7 +240,7 @@
 		"build-client-both": "npm run build-client-fallback && npm run build-client-evergreen",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || cross-env-shell NODE_ARGS=--max_old_space_size=8192 npm run build-client-both",
 		"build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
-		"build-notifications": "webpack --config='client/notifications/webpack.config.js'",
+		"build-notifications": "cb --config='client/notifications/webpack.config.js'",
```

Comment:
Keep it relative?
```suggestion
		"build-notifications": "cb --config='./webpack.config.js'",
```

Reply:
Uh won't that pick up the wrong file (root level `webpack.config.js` rather than `client/notifications/webpack.config.js`, since this is the root-level `package.json`)?

Reply:
> since this is the root-level package.json

Oh, right 🤐